client/common/directives/ModelField/ModelField.js

```diff
@@ -238,7 +238,7 @@ angular.module('dashboard.directives.ModelField', [
         template = '<label class="col-sm-2 control-label">{{ display.label || key }}:</label>\
           <div class="col-sm-10">\
             <input type="hidden" ng-model="countrycode" value="{{ display.region }}" />\
-            <input type="text" ng-model="data[key]" phone-number country-code="countrycode" ng-pattern="{{ display.pattern }}" ng-disabled="{{ display.readonly }}" ng-required="{{ model.properties[key].required }}" class="field form-control">\
+            <input type="text" ng-model="data[key]" phone-number country-code="us" ng-pattern="{{ display.pattern }}" ng-disabled="{{ display.readonly }}" ng-required="{{ model.properties[key].required }}" class="field form-control">\
```

Comment:
country-code should be dynamicly loaded based on display.region. Can you talk with @csapinoso about this. I believe that was the intention of line 240.